src/transformers/models/gptj/__init__.py

```diff
@@ -0,0 +1,50 @@
+# flake8: noqa
+# There's no way to ignore "F401 '...' imported but unused" warnings in this
+# module, but to preserve other warnings. So, don't check this module at all.
+
+# Copyright 2021 Ben Wang, Aran Komotowski, Stella Biderman, and The HuggingFace Team. All rights reserved.
```

Comment:
We usually don't add individual contributor names, I would put `mesh-transformer-jax authors, EleutherAI team and The HuggingFace Team.` cc @LysandreJik @sgugger @patrickvonplaten